gcp_variant_transforms/testing/integration/run_vcf_to_bq_tests.py

```diff
@@ -78,10 +68,9 @@ def __init__(self,
                input_pattern,
                assertion_configs,
                **kwargs):
-
-    self._name = test_name
+    # type: (TestContextManager, str, str, str, List[Dict], str) -> None
+    super(VcfToBQTestCase, self).__init__(test_name, context.project)
```

Comment:
As discussed in previous comments, I think you can (and probably should) avoid inheritance here and use composition instead.